### ISSUE TITLE
javamopagent should have a -v option

### ISSUE THREAD

**Issue by owolabileg** [runtimeverification/javamop#194]
I think that the javamop agent should have a -v option so that users can optionally see what is happening behind the scenes. This is especially useful when things fail.

**Comment by xiaohe27** [comment]
@owolabileg I see. I'll add that option soon.

**Comment by xiaohe27** [comment]
@owolabileg  I've added the -v option to javamopagent in the latest commit.

**Comment by owolabileg** [comment]
Thanks, @xiaohe27 

I tested this and noticed that the order of the -v option in the command matters. For example:

This  works well:

``javamopagent -v MultiSpec_1MonitorAspect.aj classes -n JavaMOPAgent -excludeJars``

But this fails:

``javamopagent MultiSpec_1MonitorAspect.aj classes -n JavaMOPAgent -excludeJars -v``

The exception shown is:

```
java.io.FileNotFoundException: Source '-v' does not exist
        at org.apache.commons.io.FileUtils.copyDirectory(FileUtils.java:1368)
        at org.apache.commons.io.FileUtils.copyDirectory(FileUtils.java:1310)
        at javamop.agent.SeparateAgentGenerator.generate(SeparateAgentGenerator.java:103)
        at javamop.JavaMOPAgentMain.main(JavaMOPAgentMain.java:59)
```

Not sure why this is the case, as your JCommander code looks okay at first glance. Please take a look.

**Comment by xiaohe27** [comment]
@owolabileg I've made a fix in commit `75ba0c2`
